packages/v4/src/content/accessibility/calendar-month/calendar-month.md

```diff
@@ -0,0 +1,27 @@
+---
+id: Calendar Month
+section: components
+---
+
+A **calendar month** is a box area or text that communicates and triggers user actions when clicked or selected. Calendar months are interactive elements.
+
+**Keyboard users** should be able to focus on parts of the calendar month using **Tab** to move forward and **Tab + Shift** to move backward through interactive elements. They should be able to select a focused component, such as a month on the date selection menu, using **Space** or **Enter**. They should also be able to use **Arrow Keys** to navigate through the calendar to select a date.
+
+**Screen reader users** should be able to navigate to the calendar month and it should be able to be described by a screen reader. Since a calendar month is interactive, screen readers must announce the different interactive elements of the component. For example, in the dropdown menus to select a month, screen readers must announce the month that the user is on, and within the calendar screen readers must announce the date the user is on.
+
+## Accessibility application:
+
+- Make sure the calendar month is in a position that makes sense on the tab order so that keyboard and screen reader users do not end up focusing on it out of context
```

Comment:
I meant where Calendar Month is on the page, like if it is between Button X and Button Y, users should be tabbing from Button X to the Calendar Month to Button Y. I'll remove it though since that's not something that they have to consider when implementing it!